.github/workflows/cicd.yml

```diff
@@ -389,3 +402,18 @@ jobs:
       code_coverage: false
       gather_dumps: true
       fault_injection: true
+
+  # Run the complete fault injection simulator for netebpfext in GitHub.
+  # Runs on a schedule as this takes a long time to run.
+  netebpfext_fault_injection_full:
+    needs: regular
+    if: github.event_name == 'schedule'
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      name: netebpfext_fault_injection_full
+      test_command: .\netebpfext_unit.exe
+      build_artifact: Build-x64
+      environment: windows-2019
```

Comment:
Is 2019 version intentional?
Do we need one for WS 2022 also?

Reply:
Yes it was not intentional. Updated it.